azure-pipelines.yml

```diff
@@ -71,28 +66,29 @@ steps:
     if [ $? -ne 0 ]; then exit 1; fi
   displayName: Selenium Grid Setup
 
-# - bash: |
-#     export BAPIDB__PASSWORD="$(gpitfuturesdevdbpassword)" 
-#     export ISAPIDB__PASSWORD="$(gpitfuturesdevdbpassword)"
-#     export ORDAPIDB__PASSWORD="$(gpitfuturesdevdbpassword)"
-
-#     dotnet test src/ -v n
-#   displayName: Run All Acceptance Tests
+- task: DotNetCoreCLI@2
+  displayName: 'Run dotnet restore'
+  inputs:
+    command: restore
```

Comment:
I think you'll need to add the following parameters:

```yaml
feedsToUse: config
nugetConfigPath: 'src/NuGet.config'
```

Probably worth double-checking the example path!

Reply:
This will be the issue 
``` 
(Restore target) -> 
         /home/vsts/work/1/s/src/OrderFormAcceptanceTests.Steps/OrderFormAcceptanceTests.Steps.csproj : error NU1101: Unable to find package NHSD.BuyingCatalogue.EmailClient.IntegrationTesting. No packages exist with this id in source(s): NuGetOrg
```